packages/luarocks/build.sh

```diff
@@ -1,37 +1,37 @@
 TERMUX_PKG_HOMEPAGE=https://luarocks.org/
 TERMUX_PKG_DESCRIPTION="Deployment and management system for Lua modules"
-TERMUX_PKG_LICENSE="BSD"
```

Comment:
Ah that license. Could have caused huge havoc if we were doing the same with some non-OSS stuff. Thanks for rectifying

Reply:
Do I need to do something with it?

Reply:
Nothing, I was just thanking you for fixing it :)

Reply:
Oh, Ok 👌